AAI/AAIConnectProfile.md

```diff
@@ -402,7 +403,9 @@ the Broker.
 1.  Claim Clearinghouses MUST trust at least one Broker.
 
     1.  Claim Clearinghouses MAY trust more than one Broker
-
+    
+    2.  The burden of risk assessment is on the Claim Clearinghouse to trust an access token from a Broker. RECOMMENDED that only a small number of Brokers are trusted by any one Clearinghouse in order to lessen this burden. 
```

Comment:
Not sure it has to be a small number. In the passport case, the Broker doesn't carry much risk as it just is a messenger of Visas. Trusting the Visa issuer carries most of the burden. As far as I've reasoned, limiting the brokers is mostly to help prevent rogue actors that convince researchers to use them to collect their visas, and it works because Clearinghouses are too liberal in accepting them, but meanwhile squirrels aside the visas for its own devious purpose.

By whitelisting, Brokers can be either vetted or at least removed when there is a hint of acting in bad faith.

This spec is more general than the Passport spec, so I don't want to narrow this statement too much either. The first sentence is great, but not sure what to do with the 2nd.

Reply:
Is the process for `risk assessment` out of scope of this spec or are there procedures in this doc that outline ways to mitigate risk in this specific case? I ask because `lessen this burden` makes it seem like there is already a burdensome process for this, so if the assessment is meant to be out of scope I would hesitate to say the second sentence is unnecessary.

Reply:
Yeah, maybe that's it. Risk assessment is out of scope, the spec only indicates that "trust" needs to be established.

RECOMMENDED to trust the minimum set of Brokers required to obtain the access token payload required for the resources in question?

Reply:
That sounds like a reasonable rewording. Want to convey that we know that this potentially opens risk a little and due diligence on “aud” is required.